packages/firestore/test/unit/specs/listen_spec.test.ts

```diff
@@ -1016,12 +1016,56 @@ describeSpec('Listens:', [], () => {
           // are unlistened by now.
           .userUnlistens(limitToLast)
           .client(0)
-          .expectListen(limitToLast)
+          // TODO(b/143693491) `expectListen` would also pass, which is wrong.
+          // the subsequent `expectActiveTargets` makes sure the target is
+          // removed, but we still need to fix `spec_test_runner`.
+          .expectUnlisten(limitToLast)
           .expectActiveTargets()
       );
     }
   );
 
+  specTest('Mirror Queries in primary client', [], () => {
```

Comment:
Suggest to drop "primary client" form here as this name won't be applicable to iOS and Android

Reply:
Done.